source/sync.txt

```diff
@@ -8,21 +12,129 @@
    Handle Errors </sync/error-handling>
    Add Sync to a Local-Only App </sync/local-to-sync>
+   Production Checklist </sync/production-checklist>
    Sync Production Load Testing </sync/production-load-testing>
    Event Library </sync/event-library>
+   Technical Details </sync/details>
+
+.. introduction::
+
+   You're developing a mobile app. Your users want their data saved on the cloud
+   and accessible from their other devices. Network access on a mobile device
+   can be hit or miss, so you write data locally on the device first. A
+   background process then synchronizes the data to the cloud and resolves any
+   conflicting writes.
+ 
+   {+sync+} does all that... and more!
+
+   .. button:: Start Building with a Template Starter App
+      :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
+
+.. image:: /images/plp/Spot_AzureBlue_Sync_Tech_RealmSync.png
+   :alt: Atlas Device Sync Illustration
+
+.. kicker:: What It Is
+
+Key Features
+------------
+
+.. card-group::
+   :columns: 3
+   :style: compact
+
+   .. card::
+      :headline: Realm in the Front, MongoDB in the Back
+      :icon: /images/icons/General_FEATURES_IntelligentEdge3x.png
+      :icon-alt: An icon depicting a mobile phone connecting with the cloud
+
+      {+sync+} is a bridge between client apps using the :realm:`Realm SDKs </>`
+      and a MongoDB instance running in Atlas. Realm is a lightweight database
+      optimized for mobile development.
+      
+   .. card::
+      :headline: Offline First
+      :icon: /images/icons/Technical_REALM_Offline3x.png
+      :icon-alt: An icon depicting network loss
+
+      Realm Database and {+sync-short+} seamlessly handle intermittent
+      connectivity so users can continue using your app regardless of their
+      current network status.
+
+   .. card::
+      :headline: Conflict Resolution
+      :icon: /images/icons/Technical_REALM_ConflictResolution3x.png
+      :icon-alt: An icon depicting conflict resolution
+
+      {+sync-short+} handles conflicts for you, so you don't have to write
+      complex custom code to resolve conflicting writes from multiple clients.
+      Read more: :ref:`conflict-resolution`
+
+
+.. kicker:: Where to Begin
+
+Get Started
+-----------
+
+.. card-group::
+   :columns: 2
+   :style: compact
+
+   .. card::
+      :headline: Launch a Template Starter App
+      :cta: Register for a free Atlas account
+      :url: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
+      :icon: /images/icons/Technical_REALM_iOSAndriodCrossPlatform3x.png
+      :icon-alt: An icon depicting two mobile devices synchronizing data between each other
+
+      The best way to start using {+sync-short+} is with a template starter app.
+      In the {+ui+}, under the :guilabel:`App Services` tab, you can select a
+      template. Atlas will configure a backend instance and give you the
+      frontend code for a selection of platforms. You can build your app on top
```

Comment:
Question: do we actively want to recommend building an app on top of the template? Or is it more of a learning resource so devs can see the "aha" moment faster? I worry that if we tell devs to build on this, they're going to get to breaking schema changes and then have a bad experience with the product. As an alternative, I might suggest something like:

> Explore a working example of Sync to learn how to implement it in your app.

Reply:
I think the idea is that you can take it and build whatever you want on top of it. Breaking schema changes should be something people can handle in early development - maybe we need a short guide (i.e. remove the app from the simulator, delete cluster data, restart sync)